src/modifier/modifier-type.ts

```diff
@@ -2410,8 +2421,16 @@ export class ModifierTypeOption {
   }
 }
 
+/**
+ * Calculates the team's luck value.
+ * @param party The player's party.
+ * @returns A value between 0 and 14, or 0 if the player is in Daily Run mode.
+ */
 export function getPartyLuckValue(party: Pokemon[]): integer {
-  const luck = Phaser.Math.Clamp(party.map(p => p.isAllowedInBattle() ? p.getLuck() : 0)
+  if (party[0].scene.gameMode.isDaily) {
+    return 0;
+  }
+  const luck = Phaser.Math.Clamp(party.map(p => p.isAllowedInBattle() ? (p.scene.gameMode.isDaily ? 0 : p.getLuck()) : 0)
     .reduce((total: integer, value: integer) => total += value, 0), 0, 14);
   return luck || 0;
```

Comment:
```suggestion
  return luck ?? 0;
```
Might as well change this too since you're already editing the function.